x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/add_prepackaged_rules_schema.ts

```diff
@@ -71,7 +71,11 @@ export const addPrepackagedRulesSchema = Joi.object({
     .forbidden()
     .default(true)
     .valid(true),
-  index: index.required(),
+  index: index.when('type', {
+    is: 'machine_learning',
+    then: Joi.forbidden(),
+    otherwise: Joi.required(),
+  }),
```

Comment:
Just a heads up @rylnd -- this was necessary now that we have pre-packaged rules with type `machine_learning`.